site/releases/2024-jan-17/highlights.qmd

```diff
@@ -0,0 +1,88 @@
+---
+title: "January 17, 2024"
```

Comment:
@cachafla since there is no release tag for the developer-framework repo, I did not fetch the latest notebooks and developer framework HTML output. If that is not correct, please let me know.